ci_build/view_log.py

```diff
@@ -0,0 +1,103 @@
+import re
+import json
+import os
+import sys
+
+from django.views.generic import TemplateView
+
+from community.views import get_header_and_footer
+from community.git import (
+    get_org_name,
+    get_owner,
+    get_deploy_url,
+    get_upstream_deploy_url
+)
+
+
+class BuildLogsView(TemplateView):
+    template_name = 'build_logs.html'
+
+    def get_build_info(self):
+        data = {
+            'Org name': get_org_name(),
+            'Owner': get_owner(),
+            'Deploy URL': get_deploy_url(),
+        }
+        try:
+            upstream_deploy_url = get_upstream_deploy_url()
+            data['Upstream deploy URL'] = upstream_deploy_url
+        except RuntimeError:
+            data['Upstream deploy URL'] = 'Not found'
+        return data
+
+    def get_build_logs(self, log_file_path):
```

Comment:
Add doc string for this function and for others also. Like what does this function do?

Reply:
Yeah! Sure Will add docstrings for all functions